src/rp2_common/pico_btstack/CMakeLists.txt

```diff
@@ -246,7 +248,7 @@ if (EXISTS ${PICO_BTSTACK_PATH}/${BTSTACK_TEST_PATH})
                     DEPENDS ${GATT_FILE}
                     WORKING_DIRECTORY ${GATT_PATH}
                     COMMAND ${CMAKE_COMMAND} -E make_directory ${GATT_BINARY_DIR} &&
-                            ${Python3_EXECUTABLE} ${PICO_SDK_PATH}/lib/btstack/tool/compile_gatt.py ${GATT_FILE} ${GATT_HEADER}
+                            ${Python3_EXECUTABLE} ${PICO_SDK_PATH}/lib/btstack/tool/compile_gatt.py ${GATT_FILE} ${GATT_HEADER} ${ARGN}
```

Comment:
It's a bit "clunky" passing in the arguments for the tool. It might be better if ARGN were just treated as a list of directories?

Reply:
I am sorry. I don't understand your comment. If there is a better way to achieve the same functionality so I can let the
compile_gatt.py command find my custom .gatt file, then that is the change I am requesting. This change was the
only way I could think of to do it without touching the python code, but I am no expert in CMake.

Reply:
I'm just suggesting a small change to not have to pass in the "-I" as part of the argument

Reply:
Is my new commit what you meant?

Reply:
close, I think you can just replace all that with...
list(TRANSFORM ARGN PREPEND "-I ")

Reply:
Hopefully fixed